### PR TITLE
Textfield storybook

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?
Complete the theme style of TextField complete.

TextField is consisted with `InputLabel`, `InputBase`, and `FormHelperText`. 
Originally, the height of `TextField` should be same with and without label. But for some reason, the hight of `TextField` is not equal with and without label. I guess this is the reason is in the `sass` file.

image

This PR resolves this problem by giving `top` property.

image

## Context

https://www.figma.com/file/cyyVlZJWAPEJol3zxxZ8Rp/Material-Dark?node-id=6570%3A47650

## Fleek
https://bafybeieic5ipa4ukq4fe26n2uij4vygprla3o7mbljkwgdetqmfc4v2li4.on.fleek.co/

### CODE REVIEW

File: src/components/SwapTokenInput.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { TextField } from "@mui/material"
+
+export default function SwapTokenInput(): React.ReactElement {
+  return (
+    <div>
+      <TextField name="swapTokenInput" />
+    </div>
+  )
```
Comment (ug02fast) [inline src/components/SwapTokenInput.tsx:9]: This file seems unnecessary, is there anything wrong with just using `TextField` directly in the parent component?

File: src/pages/CreatePool.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+              <Typography my={2}>{t("createPoolTypeDescription1")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription2")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription3")}</Typography>
+              <ToggleButtonGroup
+                value="basePool"
+                color="secondary"
+                size="large"
+                fullWidth
+              >
+                <ToggleButton value="usdMetaPool" size="large">
```
Comment (ug02fast) [inline src/pages/CreatePool.tsx:91]: Keep naming consistent. Here you have `usdMetaPool` and below it's `btcMetapool`. I'd suggest `metapool` as we usually refer to it as one word.

File: src/pages/CreatePool.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                  USD Metapool
+                </ToggleButton>
+                <ToggleButton value="btcMetapool">USD Metapool</ToggleButton>
```
Comment (ug02fast) [inline src/pages/CreatePool.tsx:94]: same as above

**Review by ug02fast — COMMENTED** [review]
Look good just a few questions

File: src/components/SwapTokenInput.tsx (hunk 1 above)
Comment (nkrulikovsky) [reply src/components/SwapTokenInput.tsx]: Oh, Right.
This file is not necessary in this PR.

File: src/components/AdvancedOptions.tsx [hunk 4]
```diff
@@ -174,7 +175,8 @@ export default function AdvancedOptions(): ReactElement {
               <TextField
                 type="text"
                 variant="outlined"
-                InputLabelProps={{ shrink: false }}
+                size="small"
+                // TODO check after removing SCSS
```
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply src/components/AdvancedOptions.tsx:179]: Oh, this PR is resolving this issue. Let me remove this comment.

File: src/pages/CreatePool.tsx (hunk 2 above)
Comment (nkrulikovsky) [reply src/pages/CreatePool.tsx]: Sorry. This file is not necessary on this PR.
This file is for the following PR. https://github.com/saddle-finance/saddle-frontend/pull/840

**Review by hammeiam — APPROVED** [review]

**Pull request merged**